docs/Drawer.md

```diff
@@ -295,7 +295,6 @@ The `<DrawerNavItem>` is an individual line item in the `<Drawer>`. These can be
 | classes                         | Style overrides                                                                                                                                             | `DrawerNavItemClasses`  | no       |         |
 | depth*                          | The nested depth of the item                                                                                                                                | `number`                | no       | 0       |
 | hidden                          | Sets whether to hide the nav item                                                                                                                           | `boolean`               | no       |         |
-| hidePadding                     | Sets whether to collapse the left padding when no icon is used                                                                                              | `boolean`               | no       |         |
```

Comment:
Is this intentional?  Was this prop removed from Drawer?

Reply:
It is in the sharedProps (can be passed at any level)...it's redundant here.